third_party/txt/src/txt/paragraph_txt.cc

```diff
@@ -1242,6 +1245,21 @@ void ParagraphTxt::UpdateLineMetrics(const SkFontMetrics& metrics,
       ascent = (-metrics.fAscent + metrics.fLeading / 2);
       descent = (metrics.fDescent + metrics.fLeading / 2);
     }
+
+    // Account for height_behavior in parargaph_style_.
+    //
+    // Disable first line ascent modifications.
+    if (line_number == 0 && !(paragraph_style_.height_behavior &
+                              ~TextHeightBehavior::kDisableFirstAscent)) {
```

Comment:
I don't think the `~` is producing the intended behavior.

Does `kDisableFirstAscent` signify that the `ascent = -metrics.fAscent` assignment should happen, or that it should not happen?